kword/src/commonMain/kotlin/com/mirego/trikot/kword/KWord.kt

```diff
@@ -41,9 +43,14 @@ object KWord : I18N {
         return this[key]
     }
 
-    override fun t(key: KWordKey, count: Int, vararg arguments: String): String {
-        // TODO: implement zero, one, other
-        return t(key)
+    override fun t(key: KWordKey, count: Int, vararg arguments: Pair<String, String>): String {
+        val keyWithCount = "${key.translationKey}_$count"
+        val targetString = source.get(keyWithCount)
+
+        val argumentsWithCount = arguments.toMutableSet()
+        argumentsWithCount.add(COUNT_ARGUMENT to count.toString())
+        val keyToUse = if (keyWithCount != targetString) keyWithCount else key.translationKey
```

Comment:
Couldn't we just had another method on source to return nullable ?
`source.getOptional(keyWithCount)`
Then we won't need to know that the method returns the key if not found.